io.github.FailurePoint.RandomNumberFive.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.FailurePoint.RandomNumberFive",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "random-number-five",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "random-number-five",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/FailurePoint/RandomNumberFive.git",
+                    "commit" : "212f807"
```

Comment:
this is supposed to be the full sha

Reply:
Good grief... where am I supposed to find that??

Reply:
also this one doesn't even exist

![image](https://github.com/flathub/flathub/assets/114441/6fe716e2-972d-4007-82cc-b0f82e63544f)

Reply:
well where is the full sha??

Reply:
I don't understand where I am supposed to get that.